src/views/evaluations/evaluations.js

```diff
@@ -0,0 +1,108 @@
+function EvaluationsPage(options) {
+    const newOptions = options || {};
```

Comment:
This can be done via default parameter values in ES6. Just add water... umm, actually, just add `= {}` after the `options` parameter in the function header on L1.